### PR TITLE
homerow 1.2.2 (new cask)

### PR DESCRIPTION
**Pull request by EmilioAK**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/h/homerow.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+cask "homerow" do
+  version :latest
+  sha256 :no_check
+
+  url "https://builds.homerow.app/latest/Homerow.zip"
```
Comment (khipp) [inline Casks/h/homerow.rb:5]:
```suggestion
  version "1.2.2"
  sha256 "463e46c9695857638aa529bf27b3815f2c1fd3babd8887c6e2e5dd7f2584cd02"

  url "https://builds.homerow.app/v#{version}/Homerow.zip"
```

File: Casks/h/homerow.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  name "Homerow"
+  desc "Keyboard shortcuts for every button on your screen"
+  homepage "https://www.homerow.app/"
+
```
Comment (khipp) [inline Casks/h/homerow.rb:9]:
```suggestion

  livecheck do
    url "https://www.homerow.app/appcast.xml"
    strategy :sparkle, &:short_version
  end

```

**Review by khipp — COMMENTED** [review]
Thank you for your first contribution to `homebrew-cask`, @EmilioAK! 🎉

I left a few suggestions below.

**Comment by EmilioAK** [comment]
Thanks! Didn't know that appcast url existed so good catch 😄

**Review by khipp — APPROVED** [review]
Thank you!

File: Casks/h/homerow.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  livecheck do
+    url "https://www.homerow.app/appcast.xml"
+    strategy :sparkle, &:short_version
+  end
+
+  depends_on macos: ">= :monterey"
```
Comment (bevanjkay) [inline Casks/h/homerow.rb:15]:
```suggestion
  auto_updates true
  depends_on macos: ">= :monterey"
```

I would assume based on the presence of sparkle, that this has an in-app update mechanism?

Comment (EmilioAK) [reply Casks/h/homerow.rb:15]: Yeah, you're right. It was implied before the previous addition

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**